config/ig.py

```diff
@@ -7,14 +7,28 @@
 GRAPHEMES = union(byte.LOWER, "'", "-",
                   "à", "â", "æ", "ç", "è", "é", "ê", "ë",
                   "ì", "î", "ï", "ò", "ô", "ù", "û", "ü",
-                  "ị", "ň", "ñ", "ṅ", "ọ", "ụ")
+                  "ị", "ň", "ñ", "ṅ", "ọ", "ụ", "ö")
 
 INITIAL_PUNCTUATION = utils.DEFAULT_INITIAL_PUNCTUATION
 
 FINAL_PUNCTUATION = utils.DEFAULT_FINAL_PUNCTUATION
 
 NUMERALS = byte.DIGIT
 
+TO_NEW_STANDARD_ALPHABET = cdrewrite(
```

Comment:
Could you add some comments to explain the rewrites?

Reply:
Added comments to all the language-specific rules!